hack/ci-integration.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
-go test -timeout 90m \
-  -v ./pkg/e2e \
-  -kubeconfig ${KUBECONFIG:-~/.kube/config} \
-  -args -v 5 -logtostderr \
-  "$@"
+go run ./vendor/github.com/onsi/ginkgo/ginkgo \
+    -timeout 90m \
+    -p -stream \
+    -v \
+    -failFast \
+    "$@" \
+    ./pkg/e2e/ -- --alsologtostderr -v 4 -kubeconfig ${KUBECONFIG:-~/.kube/config}
```

Comment:
Before, the tests were using `-ginkgo.noColor=true`, should that be included that here? (Does it make a difference?)